clients/privacy-center/cypress/e2e/consent-banner-gpp.cy.ts

```diff
@@ -308,9 +291,6 @@ describe("Fides-js GPP extension", () => {
         });
       });
       cy.waitUntilFidesInitialized().then(() => {
-        // TODO(PROD-1439): remove this workaround
-        workaroundGppInitializationDelayBug();
-
```

Comment:
can you also eval the `// TODO(PROD-1439): remove this workaround` on L:60 of this file?

Reply:
great catch